clusterman/kubernetes/kubernetes_cluster_connector.py

```diff
@@ -215,14 +215,13 @@ def _is_node_safe_to_kill(self, node_ip: str) -> bool:
         return True
 
     def _get_nodes_by_ip(self) -> Mapping[str, KubernetesNode]:
-        pool_label_selector = self.pool_config.read_string("pool_label_key", default="clusterman.com/pool")
-        pool_nodes = self._core_api.list_node().items
-
-        return {
-            get_node_ip(node): node
-            for node in pool_nodes
-            if not self.pool or node.metadata.labels.get(pool_label_selector, None) == self.pool
-        }
+        # TODO(https://jira.yelpcorp.com/browse/CLUSTERMAN-659)
```

Comment:
nit: this comment should look something like:

```suggestion
        # TODO(CLUSTERMAN-659): ${SHORT_SUMMARY}
```

where `${SHORT_SUMMARY}` is just a couple words about what actually needs to be done

Reply:
Noted @nemacysts I will fix this.